### PR TITLE
Add learn the concepts section: Blockchains

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
This is the PR for the Blockchains and Validators topics in the Learn the Concepts section, as outlined in the Docs Site Map gdoc. @5A11 @GataKamsky @VijaySharma815  pls have a view when possible and let me know for any change to be made. Once ready i will proceed with the editing process to add your feedback as well.

### CODE REVIEW

File: docs/learn_the_concepts/peer_to_peer_systems.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+1. **No middlemen needed**: One of the key characteristics of peer-to-peer systems is that middlemen are not always required, though they are allowed. For instance, when booking a hotel room with a travel agent, the users will be going through a middleman but if the user books a hotel room directly with the hotel itself, then this depicts a direct communication, and hence, they are communicating on a peer-to-peer basis.
+
+2. **Direct links**: direct communication between both parties can be established only in the presence of direct links between them.
+
+3. **Direct interaction**: It is not enough just for parties to be able to communicate directly, they have to be able to actually interact. Peer-to-peer systems are characterized by direct interaction. In the hotel room booking example, making a booking with the hotel with the hotel sending back a booking confirmation actually consists of a direct interaction between the two parties.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/peer_to_peer_systems.md]: You types "with the hotel" twice in a row.

File: docs/learn_the_concepts/what_is_a_blockchain.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+A central concept here is given by **mining** operations. Mining is the process by which cryptocurrency transactions between users are verified and added to the blockchain public ledger. The mining operations are also responsible for introducing new coins into the existing circulating supply. New blockchain transactions are submitted to a **pool** (i.e. **memory pool**) when these are created. A miner's responsibility is to arrange these pending transactions into blocks and confirm their authenticity. A block may be compared to a page of the blockchain ledger where many transactions are recorded, alongside with other data. More specifically, a mining node is in charge of building a candidate block from unconfirmed transactions in the memory pool. The miner will next attempt to turn this candidate block into a legitimate, confirmed block. But in order to achieve this, they must solve a challenging mathematical puzzle. This requires a lot of computational resources, but every successfully mined block will give the miner a **block reward**, consisting of **newly created cryptocurrencies** plus **transaction fees**.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/what_is_a_blockchain.md]: For now, it looks good to me! but we can write it in a simpler manner [Lame language].  

IBM Blockchain Overview documentation:
https://www.ibm.com/in-en/topics/what-is-blockchain#:~:text=Blockchain%20overview,patents%2C%20copyrights%2C%20branding).

Comment (FelixNicolaeBucsa) [reply docs/learn_the_concepts/what_is_a_blockchain.md]: I will have a look at this source as well! Thanks a lot @VijaySharma815

File: docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+## Native to ERC-20 FET
+
+This is an easy step-by-step guide explaining how to convert your native FET tokens to ERC-20 FET tokens using the Fetch Wallet.
+
+1. Create an account on the **Fetch wallet** if you do not have one, by following the instructions available at: <https://docs.fetch.ai/fetch-wallet/>. Save your 12/24 words mnemonic safely somewhere.
+    **Do not share this with anyone! If you lose this, you cannot recover your assets!**
+
+      <img src="/images/native_and_erc20/keplr_wallet.png" alt="Fetch Wallet" class="center" style="display: block; margin-left: auto; margin-right: auto;width:400px;">
```
Comment (VijaySharma815) [inline docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md]: Suggestion: Can we rename the image from "keplr_wallet.png" to "fetch_wallet.png"?

cc: @5A11

File: docs/learn_the_concepts/peer_to_peer_systems.md (hunk 1 above)
Comment (VijaySharma815) [reply docs/learn_the_concepts/peer_to_peer_systems.md]: @FelixNicolaeBucsa I think you haven't addressed this comment.

File: docs/learn_the_concepts/peer_to_peer_systems.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Another example of P2P networking is given by some online gaming platforms which adopt  a P2P structure for downloading games between users. The company [Blizzard Entertainment](https://www.blizzard.com/en-gb/) distributed Diablo III, StarCraft II, and World of Warcraft using P2P. Another large publisher, Wargaming, does the same with their World of Tanks, World of Warships, and World of Warplanes games.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/peer_to_peer_systems.md]: It's appears that you have an extra space between the words "adopt"  and "a"

File: docs/learn_the_concepts/what_is_a_blockchain.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+A **blockchain** is a series of data records that functions as a distributed, replicated digital ledger of transactions across a network of computer systems. The information in it is divided into chunks that are sorted chronologically and encrypted. When a new transaction occurs, a record of that transaction is added to every participant’s ledger. If we compare a blockchain to a conventional database, this latter one records are written in tables which might be on one or more servers in numerous different locations, but they are all centrally administered by a database administrator.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/what_is_a_blockchain.md]: @FelixNicolaeBucsa , it is: "this latter" or "the latter"?
I am not sure that's why asking.

**Review by VijaySharma815 — APPROVED** [review]
Overall Looks good to me.

File: docs/learn_the_concepts/what_is_a_blockchain.md (hunk 5 above)
Comment (FelixNicolaeBucsa) [reply docs/learn_the_concepts/what_is_a_blockchain.md]: I think both are equally correct. I can change into the latter one if you think it is preferable!

File: docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md (hunk 3 above)
Comment (FelixNicolaeBucsa) [reply docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md]: I copied and pasted the image from the already existing docs

File: docs/learn_the_concepts/peer_to_peer_systems.md (hunk 1 above)
Comment (FelixNicolaeBucsa) [reply docs/learn_the_concepts/peer_to_peer_systems.md]: Oh, i thought i did. I will modify this asap

File: docs/learn_the_concepts/what_is_a_blockchain.md (hunk 5 above)
Comment (VijaySharma815) [reply docs/learn_the_concepts/what_is_a_blockchain.md]: ignore it! I wasn't sure that's why I was confirming. LGTM

File: docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md (hunk 3 above)
Comment (VijaySharma815) [reply docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md]: Yes, this is a minor thing we can improve in the next iteration. 
I just noticed so I report it.

Comment (FelixNicolaeBucsa) [reply docs/Native_and_ERC20_FET/how_to_convert_your_fet_tokens.md]: Cool!

File: docs/learn_the_concepts/blockchains/what_is_a_blockchain.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+We can distinguish among: **public**, **private**, **consortium** and **hybrid** blockchains.  
+
+1. **Public blockchains** are entirely decentralised, permissionless, and open to everybody. Public blockchains allow all nodes to have equal access to the blockchain, the ability to add new blocks of data, and the ability to validate existing blocks of data.are largely utilised nowadays for bitcoin mining and trading. These blockchains include Bitcoin, Ethereum, and Litecoin as examples. The majority of public blockchain networks use processes known as _Proof-of-Work_ (_PoW_) or _Proof-of-Stake_ (_PoS_) to provide consensus.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/blockchains/what_is_a_blockchain.md]: ability to validate existing blocks of data.are largely utilised nowadays for  -> ability to validate existing blocks of data. are largely utilised nowadays for

File: docs/learn_the_concepts/blockchains/validators.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+Proof-of-Work Blockchains seek to generate blocks at regular intervals. PoW networks are constrained in terms of their _speed_ and _scalability_ due to how the high energy-intensive process these imply. Massive quantities of processing hash power must be used to solve the computational challenge, which uses a lot of energy. Some have called the PoW modelan environmental catastrophe because of the enormous number of blocks verified on large PoW blockchains per day. **In fact, the more intensive the validation process, the more computational power is adopted to solve the puzzle and this leads to a higher competition among validators, which equals harder proofs of work and more energy consuming operations**.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/blockchains/validators.md]: modelan is misspelt.

File: docs/learn_the_concepts/blockchains/validators.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+For instance, the Bitcoin network consumed as much power as Chile and had a carbon footprint that was equal to that of New Zealand in 2021. Technology advancement in the blockchain sector has put a lot of emphasis on addressing the negative environmental effects of crypto mining, and several alternatives have surfaced.
+
+PoW blockchains have typically offered stronger security while preserving significant decentralization, despite their speed and scalability restrictions. Due to the distributed nature of PoW systems, it would be very expensive for a bad actor to seize control of the majority of the network's computer power and take over the blockchain.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/blockchains/validators.md]: network's computer power and take over the blockchain. -> network's computing power and take over the blockchain.

**Comment by VijaySharma815** [comment]
@FelixNicolaeBucsa ,

This branch is out-of-date with the base branch
Merge the latest changes from master into this branch.

**Pull request merged**